docs/guides/latest-updates.mdx

```diff
@@ -87,6 +91,49 @@ Premium Plan users can get started right away with the IBM&reg; Circuit function
 
 This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.
 
+### 27 February 2025
+
+Early Access release of the [new IBM Quantum Platform](https://quantum.cloud.ibm.com/)
+
+In this release, Open and Pay-As-You-Go users can begin the onboarding experience, update their existing code to use the new IBM Cloud channel, and explore the platform. Refer to the [migration guide](/migration-guides/classic-iqp-to-cloud-iqp) for help.
+
+As an incentive to migrate, all users, including Open Plan, can access an IBM Quantum Heron QPU (IBM Torino) when using the new IBM Quantum Platform.
+
+<Admonition type="note">
+Premium Plan, Dedicated service, and Startup program users will receive guided migration support through their administrators in the coming months. In the meantime, they can create a free Open plan account to explore the new platform.
+</Admonition>
+
+Read more details in the [blog.](https://www.ibm.com/quantum/blog/iqp-upgrade)
+
+### 2 February 2025
+
+A new version of [dynamic circuits](/guides/classical-feedforward-and-control-flow) will start rolling out on 2 June 2025. This new version lets you run dynamic circuits at utility scale and comes with improvements in speed, quality, and usability that will gradually be added throughout the year.  Improvements include the following:
+
+- Using the Gen3 engine stack that gives a 75x speedup in execution
+- Executing branches in parallel to reduce circuit duration and improves result quality
+- Improving mid circuit measurements
+- Improving integration with dynamical decoupling
+- Improving circuit timing information visibility
```

Comment:
```suggestion
- A 75x speedup in execution by using the [Gen3 engine stack](/guides/runtime-options-overview#experimental-option-note)
- Parallel execution of branches, which reduces circuit duration and improves result quality
- Improvement in mid-circuit measurement
- Better integration with dynamical decoupling
- More visibility of circuit timing information
```

Reply:
But wouldn't it be better if these were parallel? @abbycross

Reply:
Aren't they parallel? All nouns? Also, it's how we have it in the announcement - seems less "singsongy" than having the gerunds in every bullet

Reply:
Yeah; I can see that - always fighting the passive voice, but I can live with it. :)